### ISSUE TITLE
Accessibility: Fix Postgres config page fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42723]

**Comment by codeincarnate** [comment]
Closing as this should be fixed with #43982
